source/features/deep-blame.tsx

```diff
@@ -0,0 +1,140 @@
+import './deep-blame.css';
+import blurAccessibly from 'blur-accessibly';
+import cache from 'webext-storage-cache';
+import delegate, {DelegateEvent} from 'delegate-it';
+import React from 'dom-chef';
+import versionIcon from 'octicon/versions.svg';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL, getReference} from '../libs/utils';
+
+const getPullRequestBlameCommit = cache.function(async (commit: string, prNumber: number): Promise<string | false> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			object(expression: "${commit}") {
+				... on Commit {
+					associatedPullRequests(last: 1) {
+						nodes {
+							number
+							mergeCommit {
+								oid
+							}
+							commits(last: 1) {
+								nodes {
+									commit {
+										oid
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	`);
+
+	const {nodes} = repository.object.associatedPullRequests;
+	const mergeCommit = nodes[0].mergeCommit.oid;
+	const pullRequestNumber = nodes[0].number;
+
+	if (pullRequestNumber === prNumber && mergeCommit === commit) {
+		return nodes[0].commits.nodes[0].commit.oid;
+	}
+
+	return false;
+}, {
+	cacheKey: ([commit]) => __featureName__ + commit
+});
+
+function getCommitHash(commit: HTMLElement): string {
+	return (commit.nextElementSibling! as HTMLAnchorElement).href.split('/').slice(-1)[0];
```

Comment:
```suggestion
	return (commit.nextElementSibling as HTMLAnchorElement).href.split('/').pop();
```

Let's fix it in other features as well (in this PR, if you want)